code/modules/client/preference_setup/loadout/loadout_head.dm

```diff
@@ -267,14 +278,16 @@
 
 /datum/gear/head/tcfl
 	display_name = "tcfl hat selection"
-	path = /obj/item/clothing/head/legion_beret
+	path = /obj/item/clothing/head/beret/legion
 	flags = GEAR_HAS_DESC_SELECTION
 
 /datum/gear/head/tcfl/New()
 	..()
 	var/tcfl = list()
-	tcfl["tcfl beret, dress"] = /obj/item/clothing/head/legion_beret
-	tcfl["tcfl beret, field"] = /obj/item/clothing/head/legion
+	tcfl["tcfl beret, dress"] = /obj/item/clothing/head/beret/legion
+	tcfl["tcfl beret, sentinel"] = /obj/item/clothing/head/beret/legion/sentinel
+	tcfl["tcfl beret, legate"] = /obj/item/clothing/head/beret/legion/legate
+	tcfl["tcfl peaked cap, legate"] = /obj/item/clothing/head/legion/legate
 	gear_tweaks += new/datum/gear_tweak/path(tcfl)
```

Comment:
Except the dress beret these should not go in the loadout, as they're something worn in the field.